src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/SqlPlug.java

```diff
@@ -78,7 +78,8 @@ public String description() {
         Metrics.getInstance().registerDefaultMetrics();
```

Comment:
np: should this file be named SqlPlugin vs SqlPlug?

Reply:
Good suggestion. But we can do it as part of another maintenance PR, because I am not sure about the implication of changing this name, as it is consumed by `plugin-description.properties` file, which in turn used by Elasticsearch for loading plugins.